### PR TITLE
Improve readability.

### PR DESCRIPTION
**Pull request by gabi-250**
First of all, thank you for writing this! I know little about atomic types, but your article was very approachable and informative.

This PR fixes a couple of typos. I've also rephrased several sentences I found a bit hard to parse. Let me know if you disagree with the changes.

### CODE REVIEW

**Comment by cfsamson** [comment]
Great! I think they're definitely an improvement and makes some of the sentences easier to read.

Thanks for the feedback btw. Glad you found it useful.

**Comment by gabi-250** [comment]
Thanks!
